### PR TITLE
Ported decomposer for OrderedDict from dev to 2.2 [skip ci]

### PR DESCRIPTION
**Pull request by nvidianz**

### CODE REVIEW

**Review by yhwen — APPROVED** [review]

File: tests/unit_test/fuel/utils/fobs/decomposer_test.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+        test_list = [(3, "First"), (1, "Middle"), (2, "Last")]
+        test_data = OrderedDict(test_list)
+
+        decomposer = OrderedDictDecomposer()
+        serializable = decomposer.decompose(test_data)
+        result = decomposer.recompose(serializable)
+        new_list = list(result.items())
+
+        assert test_list == new_list
```
[the remark this answers is not in the document]
Comment (nvidianz) [reply tests/unit_test/fuel/utils/fobs/decomposer_test.py:30]: Yeah. That might be a better test. Will add that to dev.

File: nvflare/fuel/utils/fobs/decomposers/core_decomposers.py [hunk 2]
```diff
@@ -42,6 +43,18 @@ def recompose(self, data: Any) -> set:
...
 
+class OrderedDictDecomposer(Decomposer):
+    @staticmethod
+    def supported_type():
+        return OrderedDict
+
+    def decompose(self, target: OrderedDict) -> Any:
+        return list(target.items())
+
+    def recompose(self, data: Any) -> set:
```
[the remark this answers is not in the document]
Comment (nvidianz) [reply nvflare/fuel/utils/fobs/decomposers/core_decomposers.py:54]: The return type is wrong. But the input should stay generic so the decomposers can use other types like tuple.

**Review by yhwen — DISMISSED** [review]

**Review by YuanTingHsieh — APPROVED** [review]

**Pull request merged**